### PR TITLE
docs(changelog): add doc for rxjs multicasting

### PR DESCRIPTION
**Pull request by subratpalhar92**
showing a pagination use case with rxjs subject

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by subratpalhar92** [comment]
Add Documentation for rxjs multicasting.

File: aio/content/guide/observables.md [hunk 1]
```diff
@@ -105,6 +105,13 @@ Notice that if you subscribe twice, there will be two separate streams, each emi
    Multicasting observables take a bit more setup, but they can be useful for certain applications. Later we will look at tools that simplify the process of multicasting, allowing you to take any observable and make it multicasting.
 </div>
 
+## Multicasting with subject
+
+A RxJS Subject is a special type of Observable that allows values to be multicasted to many Observers.
+This example shows, what should be the value of next and previous button for a pagination use cases.
+
```
Comment (jbogarthyde) [inline aio/content/guide/observables.md:112]: Something like this:

In the following example, several observers receive the same current-page value emitted by the `Subject`. Each one handles the received value differently to log the current-page value, the next-page value, and the previous-page value.

**Review by jbogarthyde — CHANGES_REQUESTED** [review]

File: aio/content/examples/observables/src/multicasting-with-subject.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Subject } from 'rxjs';
+
+// #docregion multicast_with_subject
+const pagination = new Subject<number>();
```
[the remark this answers is not in the document]
Comment (subratpalhar92) [reply aio/content/examples/observables/src/multicasting-with-subject.ts:4]: import statement is not necessary..

File: aio/content/guide/observables.md [hunk 3]
```diff
@@ -105,6 +105,13 @@ Notice that if you subscribe twice, there will be two separate streams, each emi
    Multicasting observables take a bit more setup, but they can be useful for certain applications. Later we will look at tools that simplify the process of multicasting, allowing you to take any observable and make it multicasting.
 </div>
 
+## Multicasting with subject
```
[the remark this answers is not in the document]
Comment (subratpalhar92) [reply aio/content/guide/observables.md:108]: > aking this part of the M

added h3(###)

File: aio/content/guide/observables.md [hunk 4]
```diff
@@ -105,6 +105,13 @@ Notice that if you subscribe twice, there will be two separate streams, each emi
...
+
+A RxJS Subject is a special type of Observable that allows values to be multicasted to many Observers.
```
[the remark this answers is not in the document]
Comment (subratpalhar92) [reply aio/content/guide/observables.md:110]: added the note 'In RxJS, a Subject is a special'

**Comment by subratpalhar92** [comment]
i faced git rebase issue. i am making a fresh pull request.

**Comment by subratpalhar92** [comment]
added a new pull request

https://github.com/angular/angular/pull/38479

**Pull request closed without merging**